src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -488,11 +488,12 @@ protected void writeToNBT(@NotNull final NBTTagCompound compound)
 
         //  Achievements
         @NotNull final NBTTagList achievementsTagList = new NBTTagList();
-        for (@NotNull final Achievement achievement : this.colonyAchievements)
+        for (@NotNull final Advancement achievement : this.colonyAchievements)
         {
             @NotNull final NBTTagCompound achievementCompound = new NBTTagCompound();
-            achievementCompound.setString(TAG_ACHIEVEMENT, achievement.statId);
-            achievementsTagList.appendTag(achievementCompound);
+            //todo deserialization
+            /*achievementCompound.setString(TAG_ACHIEVEMENT, achievement.);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)